src/order/compactly_generated.lean

```diff
@@ -333,4 +333,15 @@ theorem Iic_coatomic_of_compact_element {k : α} (h : is_compact_element k) :
       set.mem_empty_eq, forall_const, forall_prop_of_false, not_false_iff]⟩, },
 end⟩
 
+lemma coatomic_of_top_compact (h : is_compact_element (⊤ : α)) : is_coatomic α :=
```

Comment:
I assume there's an easy `order_iso` somewhere between `set.Iic ⊤` and the lattice itself. I think the best way to prove this would be to prove a lemma `order_iso.is_coatomic_iff` that shows that `is_coatomic` is isomorphism-invariant.